### PR TITLE
Update to support python27 runtime

### PR DESCRIPTION
**Pull request by yegle**
This is mainly for supporting python27 runtime on Google App Engine, now that the python runtime (which is python2.5) has long been deprecated.

Also added a basic test coverage. I'll be happy to take out those commits out of this PR, let me know if this looks good to you.

### CODE REVIEW

**Comment by livid** [comment]
Awesome. I'll review, test and merge before May.

Thanks a lot.

File: tpl/desktop/common/recaptcha.html [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<script type="text/javascript" src="http://{{recaptcha_args.api_server|default:'api.recaptcha.net'}}/challenge?k={{recaptcha_args.public_key}}{% if recaptcha_args.error %}&error={{recaptcha_args.error}}{% endif %}"></script>
```
Comment (yegle) [inline tpl/desktop/common/recaptcha.html]: This file needs to be removed from this commit. It was added accidentally.

File: tpl/desktop/member_settings_avatar.html [hunk 2]
```diff
@@ -21,9 +21,9 @@
     			            当前的头像
     			        </td>
     			        <td width="200" align="left">
-											{{ member|avatar:"large" }}&nbsp; 
-											{{ member|avatar:"normal" }}&nbsp;
-											{{ member|avatar:"mini" }}&nbsp;
+											{{ member|avatar:"large"|safe }}&nbsp;
```
Comment (yegle) [inline tpl/desktop/member_settings_avatar.html]: For some reason this doesn't work. I'll revert so this commit will be smaller.

**Comment by lifeidle** [comment]
我下载了v2ex-draft，尝试了在本地运行和上传gae运行都没有成功。
本地环境是App Engine Python SDK 1.9.38及Python 2.7.11
仅把app.yaml.example及config.py.example文件的.example去除了。
请问是否还要修改别处后才能运行？
万分感谢！

**Pull request closed without merging**